src/main/java/previewcode/backend/DTO/GitHubPullRequest.java

```diff
@@ -12,8 +12,7 @@
     public final Integer number;
     public final PullRequestLinks links;
 
-    private static final String TEST_URL = "https://localhost:8080/";
-    private static final String PRODUCTION_URL = "https://preview-code.com/";
+    private static final String PREVIEW_URL = "https://preview-code.com/";
```

Comment:
Why do we only have 1 now.
I would say we still should have a test url for us, or is that coming later?